### ISSUE TITLE
[bitnami/postgresql-ha] pgpool using old postgres-password key when usePasswordFile is true

### ISSUE THREAD

**Issue by lmcmipl** [bitnami/charts#20330]
### Name and Version

bitnami/postgresql-ha 11.9.4

### What architecture are you using?

None

### What steps will reproduce the bug?

This PR https://github.com/bitnami/charts/pull/13733 renamed the key from postgres-password to password but seems to have missed the case where `postgresql.usePasswordFile` is true. 

The pgpool deployment.yaml still has the key "postgresql-password" instead of "password" :

```yaml
volumeMounts:
  {{- if .Values.postgresql.usePasswordFile }}
  - name: postgresql-password
    secret:
      secretName: {{ include "postgresql-ha.postgresqlSecretName" . }}
      items:
        - key: postgresql-password
          path: pgpool-password
        - key: repmgr-password
          path: pgpool-sr-check-password
  {{- end }}
```
As a result, the pgpool pod fails to deploy with this error: 

```
kubelet            MountVolume.SetUp failed for volume "postgresql-password" : references non-existent secret key:
```

### Are you using any custom parameters or values?
```yaml
postgresql:
  usePasswordFile: true 
```

**Comment by rafariossaa** [comment]
Hi,
  Thanks for spotting this out.
  It seems to be a small change. Would you like to contribute sending a PR ? We will be glad to review and merge it.

**Comment by mouchar** [comment]
Fix provided in PR #25165
